llvm/cmake/config-ix.cmake

```diff
@@ -157,7 +157,14 @@ if(LLVM_ENABLE_ZSTD)
     find_package(zstd QUIET)
   endif()
 endif()
+
+# If LLVM_USE_STATIC_ZSTD is specified, make sure we enable zstd only if static
+# libraries are found.
+if(LLVM_USE_STATIC_ZSTD AND NOT TARGET zstd::libzstd_static)
```

Comment:
sorry what is `zstd::libzstd_static` and who sets it? it is set by find_package if it finds the static lib?

Reply:
Yes, `zstd::libzstd_static` is set during find_package if it finds static ZSTD libraries. https://github.com/intel/llvm/blob/sycl/llvm/cmake/modules/Findzstd.cmake#L60